README.md

```diff
@@ -1 +1,184 @@
-# ocw-hugo-theme
\ No newline at end of file
+# ocw-hugo-theme
+
+This is a [Hugo](https://gohugo.io/) theme implemented as a [module](https://gohugo.io/hugo-modules/).  It can be used to generate pages related to the OCW website as well as OCW course sites.
+## Local development
+
+### Dependencies
+
+- [NodeJS](https://nodejs.org/en/download/)
+- [Yarn](https://classic.yarnpkg.com/en/docs/install/#debian-stable)
+- [Go](https://golang.org/doc/install)
+- [Hugo](https://gohugo.io/getting-started/installing/)
+
+If you're running the site for the first time, or if dependencies have changed, install dependencies with:
+
+`yarn install`
+
+### Pre-load Webpack assets
+
+Additionally, you may want to run:
+
+`npm run build:webpack`
+
+Hugo is so fast that the first time you run the site, the Hugo build will happen much more quickly than the webpack build so it's best to run the build once so that assets are ready for when you start the site.  If you don't see any styling the first time you visit the page, force refresh it.  If that doesn't work, try restarting the site.
+
+### External API's
+
+This site requires running instances of [`ocw-studio`](https://github.com/mitodl/ocw-studio) and [`open-discussions`](https://github.com/mitodl/open-discussions) for some functionality.  Search results are provided by `open-discusisons` and `ocw-studio` provides some content for the home page, such as newly added courses and news items.  If you need to work with this functionality you can either run a local instance of either of these projects, or alternatively point at the RC instances and use a browser extension to temporarily disable CORS.
+
+### CORS
+
+The search page at `/search` uses the `open-discussions` search API to source results.  Running this locally and populating it with results can be tedious, so it's often easier to just point your local website at an already running version of the search API.  In order for this to work properly, you will need to use a browser extension for disabling CORS, like Moesif Origin & CORS Changer:
+
+- [Chrome](https://chrome.google.com/webstore/detail/moesif-origin-cors-change/digfbfaphojjndkpccljibejjbppifbc?hl=en-US#:~:text=Moesif%20Origin%20%26%20CORS%20Changer&text=This%20plugin%20allows%20you%20to%20send%20cross%2Ddomain%20requests.&text=This%20plugin%20allows%20you%20to%20send%20cross%2Ddomain%20requests%20directly,Allow%2DOrigin%20set%20to%20*.)
+- [Firefox](https://addons.mozilla.org/en-US/firefox/addon/moesif-origin-cors-changer1/)
+### Environment variables
+
+For development you'll need to set a few environment variables. The
+`.env.example` file will get you started.  Here are the variables and what they do:
+
+| Variable | Example | Description |
+| --- | --- | --- |
+| `SEARCH_API_URL` | `http://localhost:8063/api/v0/search/` | A URL to an `open-discussions` search API to fetch results from|
```

Comment:
Why does the theme have a dependency on `SEARCH_API_URL`? I would expect this would only be used by the site that contains `/search`

Reply:
As you've surmised, this is only necessary if you are working on changes to the search page and want to see search results in your locally running site.